api/docs/installation/localvir.md

```diff
@@ -140,6 +153,8 @@ Then, in a terminal, type
 ```sh
 (badgeyay/api) $ export FLASK_DEBUG=1
 (badgeyay/api)$ flask run
+If Flask Run does not work on Mac Try using :
+python3 -m flask run
```

Comment:
@kushthedude  `Flask run` instead of python3 -m  Flask run .

Reply:
I have written if Flask run does not work then use this

Reply:
Ok

Reply:
Use syntax to format the code